zebra/interface.c

```diff
@@ -1984,6 +2027,9 @@ static void if_dump_vty_json(struct vty *vty, struct interface *ifp,
 		json_object_string_add(json_if, "OsDescription",
 				       zebra_if->desc);
 
+	if (zebra_if->mpls)
+		json_object_boolean_add(json_if, "mplsEnabled", true);
```

Comment:
Shouldn't we add this attribute in JSON even if it's false?

Reply:
yes, sure, will do